frontend/src/scenes/cohorts/CohortFilters/CohortCriteriaRowBuilder.tsx

```diff
@@ -175,7 +175,7 @@ export interface LogicalRowDividerProps {
 export function LogicalRowDivider({ logicalOperator }: LogicalRowDividerProps): JSX.Element {
     return (
         <Divider className="logical-row-divider" orientation="left">
-            {logicalOperator}
+            <span className="text-xs text-primary-alt font-semibold">{logicalOperator}</span>
```

Comment:
I made this edit since this styling is consistent across all usages of the logicalrowdivider and I didn't want to create a new CSS file for `Surveys` just to overwrite .logical-row-divider 👀